packages/core/docs/vibe-3-changelog.md

```diff
@@ -5,6 +5,11 @@
 - `monday-ui-react-core/next` removed - import from `monday-ui-react-core` instead.
 - `SearchComponent` and `Input` components removed - use [`Search`](https://style.monday.com/?path=/docs/inputs-search--docs) and [`TextField`](https://style.monday.com/?path=/docs/inputs-textfield--docs) respectively.
 - `EditableInput` removed, use [`TextField`](https://style.monday.com/?path=/docs/inputs-editabletext--docs) instead
+- `monday-ui-react-core/types` removed, exported types are now exported from `monday-ui-react-core` [codemod]
+- `monday-ui-react-core/storybookComponents` removed, use the `vibe-storybook-components` package instead [codemod]
+- CommonJS support removed
+- `monday-ui-react-core/dist/main.css` removed, use `monday-ui-react-core/tokens` instead [codemod]
+- All components' props interfaces are exported
```

Comment:
Add about colors

Reply:
might be redundant regardless as we understood, but add it and remove all the colors related in another PR
also let's make sure once in a while that no one uses it in monolith